core/unix/rseq_linux.c

```diff
@@ -161,6 +227,7 @@ d_r_rseq_init(void)
     rseq_cs_table = generic_hash_create(GLOBAL_DCONTEXT, INIT_RSEQ_CS_TABLE_SIZE, 80,
                                         HASHTABLE_SHARED | HASHTABLE_PERSISTENT,
                                         rseq_cs_free _IF_DEBUG("rseq_cs table"));
+    rseq_check_glibc_support();
```

Comment:
For early injection (the typical upstream use case) this will never find anything and will just waste its time grabbing the module lock -- unless the app executable has glibc statically linked in and that leads to the executable exporting those __rseq_* symbols: but does that happen with a static link?

Reply:
Removed in the new approach.